extension/src/popup/basics/Card/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
```

Comment:
the SDS `Card` component should probably have a highlight variant as it's used a lot in the re-design. I'll be moving this into SDS